client/jetpack-cloud/sections/comparison/table/index.tsx

```diff
@@ -57,6 +72,17 @@ export const Table: React.FC = () => {
 							>
 								{ isFree ? translate( 'Get started' ) : getCtaLabel( item, '' ) }
 							</Button>
+
+							{ ! isFree && (
+								<ItemPrice
+									isMultiSiteIncompatible={ isMultiSiteIncompatible }
+									isIncludedInPlan={ isIncludedInPlan }
+									isOwned={ isOwned }
+									item={ item }
+									siteId={ siteId }
```

Comment:
Since the comparison page has no site context, `siteId` here can be passed as `null`

Reply:
I was thinking this, but wasn't sure. I'll change all those to null 😄

Reply:
You don't need to pass those at all, except `siteId` which can be passed as `null`

Reply:
Got that fixed 👍🏻